services/web/server/Makefile

```diff
@@ -6,7 +6,7 @@ include ../../../scripts/common-service.Makefile
 
 # overrides since it does not has same directory name
 APP_NAME := webserver
-
+PY_PACKAGE_NAME := simcore_service_webserver
```

Comment:
was this one forgotten here?

Reply:
yes. I had a script. But in any case, it does not harm since this env has to be overriden (the web-server does not follow our naming conventions and cannot be deduced automatically)